include/pybind11/functional.h

```diff
@@ -17,15 +17,16 @@ PYBIND11_NAMESPACE_BEGIN(detail)
 
 template <typename Return, typename... Args>
 struct type_caster<std::function<Return(Args...)>> {
-    using type = std::function<Return(Args...)>;
-    using retval_type = conditional_t<std::is_same<Return, void>::value, void_type, Return>;
-    using function_type = Return (*) (Args...);
+    using type          = std::function<Return(Args...)>;
+    using retval_type   = conditional_t<std::is_same<Return, void>::value, void_type, Return>;
+    using function_type = Return (*)(Args...);
```

Comment:
> ```yaml
> AlignConsecutiveAssignments: true # Varies in the source, sometimes aligned, sometimes not
> ```

Ugh, really?

I mean... there's two types of consecutive assignments: one where you are assigning bunch of similar things (like enum values, or fields of a date/time struct), and another one where you just happen to assign two things consecutively.
Somehow, it feels exaggerated or even confusing in the latter case to align them?

Reply:
If they are not related, you can add a line in between. These are all type settings, so I don't think it's that harmful to align them.

Reply:
(I'm also fine not aligned, too - no strong opinions on style, really, just want consistency and some way to automate)

Reply:
Yeah, OK, they're related (so probably extra lines are weird), but there's nothing parallel in form in the assignments (i.e., they're not all assigning 1, 2, 3, 4, ...).

But yes, maybe that's the disadvantage to the advantage of having everything automated...